index.html

```diff
@@ -7,6 +7,7 @@
     <link href="tests/lib/font-awesome.css" type="text/css" rel="stylesheet" />
     <link href="tests/lib/font-commcare.css" type="text/css" rel="stylesheet" />
     <link href="tests/lib/hqstyle-core.css" type="text/css" rel="stylesheet"></link>
+    <link href="bower_components/codemirror/lib/codemirror.css" type="text/css" rel="stylesheet" />
```

Comment:
This is already included in the build file, so it will get minified with other package's css. Previously this lived in src/main.js, but since we can no longer use paths for codemirror I needed to pull this out